### ISSUE TITLE
issue with comma being inserted after parens

### ISSUE THREAD

**Issue by dperrymorrow** [ThisIsManta/stylus-supremacy#52]
Hi great project, enjoying using thus far. 
However I am Having an issue with stylus supremacy that Im trying to sort out.

```stylus
td
  padding: ($data.form-padding / 2) $data.form-padding
```

After formatting, a comma has been inserted

```stylus
td
  padding: ($data.form-padding / 2), $data.form-padding
```

which compiles to invalid CSS

```css
td {
    padding: 7px,14px
}
```
any help would be much appreciated, It seems to have to do with the parens / calculation

**Comment by dperrymorrow** [comment]
further clarification, this does not occur when using the command line, but does in both Atom and VsCode

**Comment by ThisIsManta** [comment]
Hello, thank you for reporting the issue. I'll have a look this weekend. I've been very busy at work :p

**Comment by dperrymorrow** [comment]
Thanks for your quick reply. I have added a failing test here if that helps https://github.com/dperrymorrow/stylus-supremacy/commit/959b9f5e684e172edb823a041400261e24d30df5

**Comment by ThisIsManta** [comment]
This will be fixed in v2.14.2 or newer.
